lib/Sentry.js

```diff
@@ -90,6 +94,13 @@ export class Sentry {
         if (Sentry.isNativeClientAvailable()) Sentry._nativeClient.nativeCrash();
     }
 
+    static setDataCallback(callback) {
+      Sentry._ravenClient.setDataCallback(callback);
+      if (Sentry.isNativeClientAvailable()) {
```

Comment:
I would remove this `if` completely, we don't need this on the native client since the event gets mutated before its sent to the native code.